src/platform/scene/feature-toggles.ts

```diff
@@ -7,10 +7,10 @@ export type PortableExperiencesToggles = EnabledDisabled | 'hideUi'
 export const portableExperiencesToggles: PortableExperiencesToggles[] = [...toggles, 'hideUi']
 
 /** @alpha */
-export type FeatureToggles = {
+export type FeatureToggles = Partial<{
```

Comment:
won't be better to mark each field as optional? otherwise you won't be able to have mandatory fields

Reply:
good idea